### ISSUE TITLE
Gameover Records with expired links

### ISSUE THREAD

**Issue by zapplecat** [JessicaNgo/TeleGiphy#104]
As per #41, it seems like giphy links expire. Maybe we should have a static gif for "expired" links when we can't get the actual gifs anymore.

**Comment by corez92** [comment]
Did those two links in that OP ever work though?

The ones I posted are still up. The url structure for them are different from yours, dunno if that's why

**Comment by zapplecat** [comment]
Yeah, they used to work. No longer though.

**Comment by corez92** [comment]
Honestly I never really understood why people would ever wanna check out gameover records after finishing a game. After like 1 minute people just don't care anymore or at least I wouldn't

**Comment by zapplecat** [comment]
Yeah, technically not terribly important. Gameover Records are there so that if a game ends, people can still get the results (since the game has ended and been deleted).
